ssl/ssl_lib.c

```diff
@@ -3712,7 +3712,7 @@ void SSL_set_not_resumable_session_callback(SSL *ssl,
 
 /*
  * Allocates new EVP_MD_CTX and sets pointer to it into given pointer
- * vairable, freeing EVP_MD_CTX previously stored in that variable, if any.
+ * variable, freeing EVP_MD_CTX previously stored in that variable, if any.
  * If EVP_MD pointer is passed, initializes ctx with this md Returns newly
```

Comment:
I don't need to put a finger in this.. ;)
A grep on codebase show that it's nearly a draw ( match nul ...).